### PR TITLE
Created K8 namespaces and altered deployment files 

### LINKED ISSUE #44: Create Kubernetes MetaData Files and Namespaces for Deployment
**As a** Developer
**I need** to create K8S Metadata files and Namespaces
**So that** I can deploy the application to K8S 

**Details & Assumptions:** 
* The following metadata files need to be present to ensure successful deployment
   - deployment.yaml
   - service-dev.yaml
   - service-prod.yaml
   - postgres.yaml
* A dev namespace and a prod namespace should be created in the Kubernetes cluster to deploy the development and production versions of the promotions service

**Acceptance Criteria:**

```gherkin
Given The K8S Cluster is in an available state 
When I run the command `kubectl -n NAMESPACE apply -f METADATA_FILENAME` 
Then the corresponding cluster resources should be deployed properly in that namespace
When I run the command `kubectl get -n NAMESPACE <resource> <resource-name>`
Then the command should return the resource in a running state confirming that the resource has been deployed properly
```

### PR DESCRIPTION
**Pull request by Aman-Chopra**
Created namespaces and altered deployment files to deploy the service in dev and prod namespaces.

### CODE REVIEW

File: deploy/postgresql.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ spec:
   ports:
     - port: 5432
       targetPort: 5432
-      nodePort: 31032
+      # nodePort: 31032
```
Comment (Black-Caesaris) [inline deploy/postgresql.yaml:59]: Should we rather separate this service into prod, and dev services for Postgres? Instead of commenting out this line.

**Review by Black-Caesaris — COMMENTED** [review]
One small doubt on commenting out Postgres service port.

**Comment by Aman-Chopra** [comment]
> One small doubt on commenting out Postgres service port.

Since it’s a DB, we don’t need to expose a port for this (also mentioned by Professor in one of my threads in #troubleshot) Of course in a real-world scenario, it would be better to separate this out. (As per need)

File: deploy/postgresql.yaml (hunk 1 above)
Comment (Aman-Chopra) [reply deploy/postgresql.yaml:59]: Let it be this way for now and we can do that in the future depending on the requirements!

**Review by adwalvekar — APPROVED** [review]

**Review by Black-Caesaris — APPROVED** [review]
LGTM

**Pull request merged**